client/devdocs/design/search-collection.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+let FilterSummary, Hider;
```

Comment:
These aren't truly "Internal dependencies" in the sense of modules imported from elsewhere in the app. Furthermore, we're not as concerned about declaring variables at the top with `let` and `const`.

You could remove the "Internal dependencies" block altogether, and declare the variables at the point of assignment, e.g. `const FilterSummary = ...`